cluster/ci/config/prow/plugins.yaml

```diff
@@ -1,4 +1,26 @@
 plugins:
+  cri-o/cri-o:
```

Comment:
If we did an org hook can we make this org level and only `approve` at the repo level?

Reply:
I don’t know yet if we want to add all this automation to every repo yet. Would involve a bigger conversation with the cri-o team